src/Modules/Events/EventDetailsContainer.js

```diff
@@ -15,7 +15,7 @@ const EventDetailsContainer = (props) => {
 
   const { id: eventDateId } = useParams();
   const [isLoading, setLoading] = useState(false);
-  const [setUserToken] = useState(undefined);
+  const [userToken, setUserToken] = useState(undefined);
```

Comment:
https://github.com/midohiofoodbank/freshtrak-client/pull/139 Issue is fixed here.